### PR TITLE
2419 Minimum TLS version cluster config property

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2419. Dependency on https://github.com/redpanda-data/docs/pull/641

## Page previews

[What's new](https://deploy-preview-639--redpanda-docs-preview.netlify.app/24.2/get-started/whats-new/#minimum-tls-version)
[Configure Kafka TLS Encryption](https://deploy-preview-639--redpanda-docs-preview.netlify.app/24.2/manage/security/encryption/)

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/pages/security/encryption.adoc [hunk 1]
```diff
@@ -326,6 +326,24 @@ Schema Registry and HTTP Proxy connect to Redpanda over the Kafka API. If you co
...
 
+=== Manage the minimum TLS version
+
+Redpanda sets the minimum TLS version for all clusters to 1.2, using the xref:reference:properties/cluster-properties.adoc#tls_min_version[`tls_min_version`] cluster configuration property. This prevents client applications from negotiating a downgrade to the TLS version when they make a connection to a Redpanda cluster.
+
+You can update the minimum TLS version of your clusters to `v1.0`, `v1.1` or `v1.3` using `rpk`. Replace the placeholder in brackets. 
+
+[,bash]
+----
+rpk cluster config set tls-min_version=<version-number>
```
Comment (asimms41) [inline modules/manage/pages/security/encryption.adoc]: Are these commands correct (I didn't get a chance to test them) and would you change the min version at this point in the setup?

Comment (michael-redpanda) [inline modules/manage/pages/security/encryption.adoc]:
```suggestion
rpk cluster config set tls_min_version <version-number>
```

version number can be:
* `v1.0`
* `v1.1`
* `v1.2`
* `v1.3`

**Review by michael-redpanda — CHANGES_REQUESTED** [review]

**Review by deniscoady — APPROVED** [review]
LGTM!

File: modules/manage/pages/security/encryption.adoc (hunk 1 above)
Comment (asimms41) [reply modules/manage/pages/security/encryption.adoc]: Updated.

Comment (asimms41) [reply modules/manage/pages/security/encryption.adoc]: @michael-redpanda. The command is now updated and all versions are listed on line 333. Is that OK?

File: modules/manage/pages/security/encryption.adoc [hunk 2]
```diff
@@ -326,6 +326,24 @@ Schema Registry and HTTP Proxy connect to Redpanda over the Kafka API. If you co
...
 
+=== Manage the minimum TLS version
+
+Redpanda sets the minimum TLS version for all clusters to 1.2, using the xref:reference:properties/cluster-properties.adoc#tls_min_version[`tls_min_version`] cluster configuration property. This prevents client applications from negotiating a downgrade to the TLS version when they make a connection to a Redpanda cluster.
+
+You can update the minimum TLS version of your clusters to `v1.0`, `v1.1` or `v1.3` using `rpk`. Replace the placeholder in brackets. 
+
+[,bash]
+----
+rpk cluster config set tls-min_version<version-number>
```
Comment (michael-redpanda) [inline modules/manage/pages/security/encryption.adoc]:
```suggestion
rpk cluster config set tls_min_version <version-number>
```

**Review by michael-redpanda — CHANGES_REQUESTED** [review]

**Review by michael-redpanda — APPROVED** [review]

File: modules/get-started/pages/whats-new.adoc [hunk 3]
```diff
@@ -15,6 +15,10 @@ Redpanda now provides a FIPS 140-compliant binary to support running Redpanda in
 
 Redpanda's xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry] now supports JSON serialization format, in addition to Avro and Protobuf. You can use a https://json-schema.org/[JSON Schema^] format to validate messages and ensure compatibility between schema versions.
 
+== Minimum TLS version
```
Comment (JakeSCahill) [inline modules/get-started/pages/whats-new.adoc]: Maybe we should group this with certificate revocation lists under a parent heading of TLS enhancement?

```
== TLS enhancement

* *Minimum TLS version*: For enhanced security, Redpanda now has a cluster configuration property to xref:manage:security/encryption.adoc#manage-the-minimum-tls-version[control the minimum TLS version] to use for encryption. By default, xref:reference:properties/cluster-properties.adoc#tls_min_version[`tls_min_version`] is set to `v1.2`, which prevents client applications from using older TLS versions to connect to your clusters.
* *Certificate revocation lists*: When enabling TLS encryption for the Kafka, Admin, HTTP Proxy or Schema Registry APIs, you can now xref:manage:security/encryption.adoc[add a certificate revocation list (CRL)] to your configuration. Redpanda uses the CRL to check and reject connections from entities using certificates already revoked by a certificate authority.
```

**Review by JakeSCahill — APPROVED** [review]
LGTM.

I left one suggestion but it's not a blocker.

**Pull request merged**